polyfactory/factories/base.py

```diff
@@ -184,10 +184,14 @@ class BaseFactory(ABC, Generic[T]):
     _factory_type_mapping: ClassVar[dict[Any, type[BaseFactory[Any]]]]
     _base_factories: ClassVar[list[type[BaseFactory[Any]]]]
 
+    _providers: ClassVar[dict[Any, Callable[[], Any]]]
+    "List of type providers that apply to all factories"
+
     # Non-public attributes
     _extra_providers: dict[Any, Callable[[], Any]] | None = None
+    "Used to copy providers from once base factory to another dynamically generated factory for a class"
```

Comment:
```suggestion
    """Used to copy providers from once base factory to another dynamically generated factory for a class"""
```
update to match format of other class doc